frontend/components/feedback-task/footer/Pagination.component.vue

```diff
@@ -41,21 +42,34 @@ export default {
       type: Number,
       default: () => 1,
     },
-    nextButtonMessage: {
-      type: String,
-      default: () => "Next",
+    items: {
+      type: Number,
+      default: () => 0,
     },
-    prevButtonMessage: {
-      type: String,
-      default: () => "Prev",
+    itemsPerPage: {
+      type: Number,
+      default: () => 1,
     },
   },
   computed: {
     isFirstPage() {
       return this.currentPage === 1;
     },
-    pageFromRoute() {
-      return parseFloat(this.$route.query?._page) || 1;
+    totalPages() {
+      return Math.ceil(this.items / this.itemsPerPage);
+    },
+    currentPaginationRange() {
```

Comment:
I think we need to do a PageCriteria to support this, because, now the pagination could be modify by the user